docs/lib/datastore/fragments/android/getting-started/40_codegen.md

```diff
@@ -0,0 +1,2 @@
+<!-- TODO: add Android gradle tools codegen instructions -->
+<!-- shorter version of the one in the tutorial -->
```

Comment:
@jpignata can you help on this? For iOS I used a simplified version of what's described in the Tutorial (no screenshots, textual description of the required steps)